src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -61,6 +79,13 @@ public void testDoPost_NewUser() throws IOException, ServletException {
     Mockito.when(mockUserStore.isUserRegistered("test username")).thenReturn(false);
     registerServlet.setUserStore(mockUserStore);
 
+				UUID fakeConversationId = UUID.randomUUID();
+    Conversation fakeConversation =
+        new Conversation(fakeConversationId, UUID.randomUUID(), "test_conversation", Instant.now());
+    Mockito.when(mockConversationStore.getActFeedConversation())
+        .thenReturn(fakeConversation);
+
+
```

Comment:
hmm I'm having a hard time understanding this part, why do we need the fakeConversation? since you didn't modify doPost() for RegisterServlet.java right?

Reply:
I actually forgot to talk about my most important modification to registerservlet.java in my opening comments!

I actually modified the doPost() for RegisterServlet.java so that a new message is created and written through to the datastore/messageStore whenever a new user is successfully created. This fakeConversation is created so that the test can return this conversation when .getActFeedConversation() is called on the mockConversationStore object. Normally the activity feed conversation is used to give the new message the correct conversationID so that it can be retrieved later for the activity feed.

Reply:
oh that makes sense! thanks for the explanation!